modules/ossm-rn-fixed-issues.adoc

```diff
@@ -14,20 +14,41 @@ Provide the following info for each issue if possible:
+//current release is 2.5.1/2.4.7/2.3.11 scheduled for April 18, 2024
 
-* https://issues.redhat.com/browse/OSSM-1397[OSSM-1397] Previously, if you removed the `maistra.io/member-of` label from a namespace, the {SMProductShortName} Operator did not automatically reapply the label to the namespace. As a result, sidecar injection did not work in the namespace.
+* https://issues.redhat.com/browse/OSSM-6177[OSSM-6177] Previously, when validation messages were enabled in the `ServiceMeshControlPlane` (SMCP), the `istiod` crashed continuously unless `GatewayAPI` support was enabled. Now, when validation messages are enabled but `GatewayAPI` support is not, the `istiod` does not continuously crash.
+
+* https://issues.redhat.com/browse/OSSM-6163[OSSM-6163] Resolves the following issues:
 +
-The Operator would reapply the label to the namespace when you made changes to the `ServiceMeshMember` object, which triggered the reconciliation of this member object.
+** Previously, in the {SMProductShortName} control plane (SMCP) v2.5, an unstable Prometheus image was included and users were not able to access the Prometheus dashboard. Now, in the {SMProductShortName} operator 2.5.1, the Prometheus image has been updated.
 +
-Now, any change to the namespace also triggers the member object reconciliation.
+** Previously, in the {SMProductShortName} control plane (SMCP), a Grafana data source was not able to set Basic authentication password automatically and users were not able to view metrics from Prometheus in Grafana mesh dashboards. Now, a Grafana data source password is configured under the `secureJsonData` field. Metrics are displayed correctly in dashboards.
+
+* https://issues.redhat.com/browse/OSSM-6148[OSSM-6148] Previously, the {SMPlugin} did not respond when the user clicked on any option in the menu of any node on the **Traffic Graph** page. Now, the plugin responds to the selected option in the menu by redirecting to the corresponding details page.
+
+* https://issues.redhat.com/browse/OSSM-6099[OSSM-6099] Previously, the {SMPlugin} failed to load correctly in an IPv6 cluster. Now, the OSSMC plugin configuration has been modified to ensure proper loading in an IPv6 cluster.
+
+* https://issues.redhat.com/browse/OSSM-5960[OSSM-5960] Previously, the {SMPlugin} did not display notification messages such as backend errors or Istio validations. Now, these notifications are displayed correctly at the top of the plugin page.
+
+* https://issues.redhat.com/browse/OSSM-5959[OSSM-5959] Previously, the {SMPlugin} did not display TLS and Istio certification information in the **Overview** page. Now, this information is displayed correctly.
+
+* https://issues.redhat.com/browse/OSSM-5902[OSSM-5902] Previously, the {SMPlugin} redirected to a "Not Found Page" error when the user clicked on the **Istio config** health symbol on the **Overview** page. Now, the plugin redirects to the correct **Istio config** details page.
```

Comment:
ISG: Avoid phrasal verbs when you can do so without changing meaning. 

"clicked on" -> "clicked"